### PR TITLE
Twitter: display latest tweet

### LINKED ISSUE #3324: Twitter: Latest tweet not being displayed
I noticed the latest tweet is not being displayed as per the [instant answer description](https://duck.co/ia/view/twitter), instead the channel description is displayed.

Example query: _southwest airlines twitter_ or _@southwestair_
Query URL: https://duckduckgo.com/?q=%40southwestair&t=ffab&ia=twitter

twitter_ia

------
IA Page:  http://duck.co/ia/view/twitter 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @jagtalon

### PR DESCRIPTION
**Pull request by willyb321**
Twitter: display latest tweet

## Description of new Instant Answer, or changes

Make `twitter x`  or `@x` queries display the latest tweet.

## Related Issues and Discussions

Fixes #3324 

## People to notify

@radioactive73
@pjhampton 

---

Instant Answer Page: https://duck.co/ia/view/twitter

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by kirkins** [comment]
Checked out branch and tested. Working and LGTM 👍

**Comment by moollaza** [comment]
@willyb321 thanks for this PR. 

TL;DR -- Unfortunately we can't use this. Sorry but I'm closing the PR.

Reason:
Currently we're nearly maxing out the number of Twitter API requests we're allowed. We've even gone over a few times. If we switch this IA to show the latest tweet, by default it will cache the result for 24 hours, so this IA would quickly become stale for active accounts. We can't reduce the current caching time any because then we'll definitely go over the API limits.

We've been considering moving to an enterprise API, but from what I've seen that will require OAUTH which can't be implemented in the open source repos. We haven't had any negative feedback about the current IA, but in some cases it's unclear what the user wants. We have plans to improve the results around social profiles, but it's not an immediate focus.

I hope this makes sense?

Thanks again and sorry I can't merge this!

**Comment by willyb321** [comment]
@moollaza Yeah that makes sense. Fair enough, I can get why that wouldn't work.
